src/guide/jbs-jdk-bug-system.md

```diff
@@ -125,7 +125,7 @@ Add a comment when adding a (Rel)[-wnf]{.jbs-label} label so that it's clear for
 1. Issue relevant to JDK 8 and all future releases.
 2. No need to add additional releases as they are implied.
 3. Adding the occasional LTS release value is ok.
-4. Use N-na to indicate that the issue is no longer relevant from that release, which could be due to the feature or platform being removed or the code being rewritten.
+4. Use N-na to indicate that the issue is no longer relevant from that release, which could be due to the feature or platform being removed or the code has been rewritten.
```

Comment:
I think that "being rewritten" was correct since it parallels the earlier "being removed".

Reply:
Agree with Iris.

Reply:
reverted the change